src/components/Calculator.jsx

```diff
@@ -1,77 +1,99 @@
 import React, { Component } from "react";
+import calculate from "../logic/calculate";
 
 class Calculator extends Component {
```

Comment:
According to the requirements:

![image](https://user-images.githubusercontent.com/57421823/170459419-a801f303-c17a-49ec-ab03-dd4228df6ab9.png)

Therefore:

- [ ] Please, kindly use lifecycle methods, lift the state up, and pass props to at least one child component.

## Explanation

Your project is not complete yet.  We need to practice what we have learned until now, according to the requirements.

## Hints

- You will need to create at least one child component.  (e.g. a `<Button />` component).
- Lifecycle methods include componentDidMount(), componentDidUpdate(), componentWillUnmount(), among others.

Reply:
Thanks for the review!